cmake/psi4OptionsTools.cmake

```diff
@@ -197,21 +197,6 @@ macro(option_with_flags option msg default)
     endif()
 endmacro()
 
-#Macro so I don't have to look at a ton of if statements for adding each plugin
-#
-#Syntax: optional_plugin(<plugin name>)
-#
-macro(optional_plugin plugin_name)
-string(TOUPPER ${plugin_name} PLUGIN_NAME)
-if(${ENABLE_${PLUGIN_NAME}})
-   find_package(${plugin_name} REQUIRED)
-   set_property(GLOBAL APPEND PROPERTY PSI4_MODULES ${${PLUGIN_NAME}_LIBRARIES})
-   add_definitions(-DENABLE_${PLUGIN_NAME})
-else()
-   add_library(${plugin_name} INTERFACE)
-endif()
-endmacro(optional_plugin plugin_name test_names)
```

Comment:
check with @amjames just to make sure this wasn't the pasture plugin treatment. Not that I think any code is coming back from the pasture.

Reply:
Nope, I set up the pasture plugins to build as if they were standard plugins. They just use the `add_psi4_plugin` macro. They are built as downstream and loaded in the driver by specific request, if they have been installed into the psi4 install. 

All that said I just took look a quick look at the pasture CMake, I have no idea if those plugins are even still working. We have been relying on people to report a problem if one comes up, so either they are working fine or no one is using them. There is no build testing done on that repo as far as I know, and they are not included in conda pkgs or anything.